integration/run/run_test.go

```diff
@@ -1157,6 +1158,51 @@ func TestAppWithBadRegion(t *testing.T) {
 	}
 }
 
+func TestAppWithBadDefaultRegion(t *testing.T) {
+	helper.StartController(t)
+
+	ctx := helper.GetCTX(t)
+	kclient := helper.MustReturn(kclient.Default)
+	c, ns := helper.ClientAndNamespace(t)
+
+	storageClasses := new(storagev1.StorageClassList)
+	err := kclient.List(ctx, storageClasses)
+	if err != nil || len(storageClasses.Items) == 0 {
+		t.Skip("No storage classes, so skipping VolumeBadClassInImageBoundToGoodClass")
```

Comment:
looks like the wrong test name here

```suggestion
		t.Skip("No storage classes, so skipping TestAppWithBadDefaultRegion")
```

Reply:
Thanks for catching this!